app.py

```diff
@@ -72,6 +72,17 @@ def armstrong():
     print(n,"is an Armstrong number")
   else:
     print(n,"is not an Armstrong number")
+    
+  def palin_pyramidpattern(): 
+    
+    n = int(input("Enter the number of rows :"))
+    for i in range(1, n+1):
+     for k in range(1, i+1):
+       print(k, end = " ")
+    for l in range(i-1, 0, -1):
+      print(l , end = ” “)
```

Comment:
Can we make this more efficient @SunandanChakrabarti ?
This is a very basic way to do it and can be improved if thought more!